ios/RCTConvert+BitmovinPlayer.swift

```diff
@@ -1156,7 +1156,8 @@ extension RCTConvert {
 
         return RNPlayerViewConfig(
             uiConfig: rnUiConfig(json["uiConfig"]),
-            pictureInPictureConfig: pictureInPictureConfig(json["pictureInPictureConfig"])
+            pictureInPictureConfig: pictureInPictureConfig(json["pictureInPictureConfig"]),
+            hideFirstFrame: json["hideFirstFrame"] as? Bool ?? false
```

Comment:
I would make `hideFirstFrame` as an optional bool (`Bool?`), so we can indicate we want the default SDK value, indicated by `nil`.
This way we don't have to sync the default value both here in RN and on the native SDK.

That way we would only modify this flag when we have a known value.

What do you think?

Reply:
Makes definitely sense. Done in c833751b (although I'm not 100% sure that's how it's done in Swift, but it seems to work)/